shell/browser/api/electron_api_web_contents.cc

```diff
@@ -1473,6 +1473,15 @@ void WebContents::HandleNewRenderFrame(
     web_frame->Connect();
 }
 
+void WebContents::OnBackgroundColorChanged() {
+  absl::optional<SkColor> color = web_contents()->GetBackgroundColor();
+  if (color.has_value() && color.value() == SK_ColorTRANSPARENT) {
```

Comment:
Why only when transparent? Should we forward this for all color changes?

Reply:
yes, we can - initially i just scoped it to transparent since that is the issue directly affected and at no point during testing was i able to observe a similar issue with other colors set via css

Reply:
What about partially-transparent colors e.g. `rgba(255,0,0,0.5)`?

Reply:
fixed up - set for all color changes now.